server/canvas/api.py

```diff
@@ -0,0 +1,60 @@
+import requests
+
+from server.extensions import cache
+
+def canvas_api_request(canvas_course, method, endpoint, **kwargs):
+    url = 'https://' + canvas_course.api_domain + '/api/v1' + endpoint
+    headers = {
+        'Accept': 'application/json',
+        'Authorization': 'Bearer ' + canvas_course.access_token,
+    }
+    response = requests.request(method, url, headers=headers, **kwargs)
+    response.raise_for_status()
```

Comment:
Does this mean we'll 500 if the canvas API ever does anything unexpected? It's similar to how we used to (or currently respond) the autograder. 

It's not a big deal - I'm okay only changing this behavior if see lots of errors from the canvas API